### DISCUSSION TITLE
Where to hook in custom javascript when nas track changes?

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#1294] [General]
As per the title I want to hook in some of my own javascript everytime a new track begins to play from my nas library.

But I get confused trying to find my bearings in the code. I just can't seem to orient myself.

Can someone give me a hint of where I could enter a couple of lines of javascript that gets executed every time a new song starts to play?

Thanks!

**Comment by kan-ibal** [comment]
Have a look at /srv/http/bash/status-push.sh and generated /srv/http/data/shm/status file.

**Comment by kreativkod** [comment]
@kan-ibal 
Thanks for your suggestion. 
I've been playing on and off with this for a few hours total now and I just cannot penetrate the logic. I probably have to abort my project an file a feature request instead. :-)
